internal/cmd/version_command.go

```diff
@@ -50,7 +65,12 @@ func (c *VersionCommand) Run(args []string) int {
 		return 0
 	}
 
-	c.Ui.Output(string(c.Version))
+	ver := string(c.Version)
+	if output.BuildGoVersion != "" && output.BuildGoOS != "" && output.BuildGoArch != "" {
+		ver = fmt.Sprintf("%s\ngo%s %s/%s", c.Version, output.BuildGoVersion, output.BuildGoOS, output.BuildGoArch)
```

Comment:
As mentioned, the existing output format may be used by other tools which parse it (because they haven't yet switched to parsing the JSON output) and may expect the first line to only have the version, so perhaps we can avoid at least _some_ breaking changes by moving the additional fields below? e.g.

```
0.11.0
go: go1.15.2
platform: darwin/amd64
compiler: gc
```